lib/api/model/model.dart

```diff
@@ -275,80 +286,79 @@ class ZulipStream {
   Map<String, dynamic> toJson() => _$ZulipStreamToJson(this);
 }
 
-/// As in `subscriptions` in the initial snapshot.
+/// Policy for which users can post to the stream.
 ///
-/// For docs, search for "subscriptions:"
-/// in <https://zulip.com/api/register-queue>.
-@JsonSerializable(fieldRename: FieldRename.snake)
-class Subscription {
-  // First, fields that are about the stream and not the user's relation to it.
-  // These are largely the same as in [ZulipStream].
-
-  final int streamId;
-  final String name;
-  final String description;
-  final String renderedDescription;
-
-  final int dateCreated;
-  final int? firstMessageId;
-  final int? streamWeeklyTraffic;
-
-  final bool inviteOnly;
-  final bool? isWebPublic; // TODO(server-??): doc doesn't say when added
```

Comment:
Also
    changed `isWebPublic` to be non-null. It has been a
    field on subscriptions since at least FL 8 (see
    zulip/zulip@f93c19ec6 in `zerver/openapi/zulip.yaml`
    and `gather_subscriptions_helper` in `zerver/lib/actions.py`).

Thanks. I don't understand the last bit, though — what specifically should the reader be looking at in that function? (It's a long function; and searching there for `is_web_public` doesn't find anything.)

Reply:
It was a bit of a journey tracking it down, the openapi docs shows the existence of `is_web_public` on subscriptions but on the `/users/me/subscriptions` endpoint. To confirm it's in the data bundle from `register` the journey went from `zerver.views.events_register.events_register_backend` -> `zerver.lib.events.do_events.register` -> `zerver.lib.events.fetch_initial_state_data` and from there tracking down what populates `subscriptions`.  I've amended the comment with more context.